build.gradle

```diff
@@ -92,8 +92,22 @@ subprojects {
     tasks.check.dependsOn(tasks.jacocoTestReport)
     tasks.check.dependsOn(javadoc)
 
-    if (System.env.CIRCLE_TEST_REPORTS) {
-        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
+    if (System.env.CIRCLECI) {
+        test.reports.junitXml.destination = new File(System.env.TESTS_DIR, it.getName())
+
+        task copyCiArtifactsToCircle {
```

Comment:
glob seems like a fragile/bad pattern to start. What happens when you make a `scripts/` dir, or put utility projects in an `extra/` subdir (or use some codegen thing that makes sub-projects).